schemas/sublime-settings.json

```diff
@@ -1,7 +1,793 @@
+  "type": "object",
+  "properties": {
+    "color_scheme": {
+      "description": "Sets the colors used within the text area.",
+      "type": "string",
+      "pattern": "^.+\\.(sublime-color-scheme|tmTheme)$",
+      "default": "Monokai.sublime-color-scheme"
+    },
+    "font_face": {
+      "description": "Font face of the text area.\n`font_face` is overridden in the platform specific settings file, for example, \"Preferences (Linux).sublime-settings\".\n Because of this, setting it here will have no effect: you must set it in your User File Preferences.",
+      "type": "string",
+      "default": ""
+    },
+    "font_size": {
+      "description": "Point size of the font used in the text area.\n`font_size` is overridden in the platform specific settings file, for example, \"Preferences (Linux).sublime-settings\".\n Because of this, setting it here will have no effect: you must set it in your User File Preferences.",
+      "type": "integer",
+      "default": 10
+    },
+    "font_options": {
+      "description": "Valid options on all platforms are:\n  - \"no_bold\": Disables bold text\n  - \"no_italic\": Disables italic text\nAntialiasing options:\n  - \"no_antialias\": Disables antialiasing\n  - \"gray_antialias\": Uses grayscale antialiasing instead of subpixel\nLigature options:  - \"no_liga\": Disables standard ligatures (OpenType liga feature)\n  - \"no_clig\": Disables contextual ligatures (OpenType clig feature)\n  - \"no_calt\": Disables contextual alternatives (OpenType calt feature)\n  - \"dlig\": Enables discretionary ligatures (OpenType dlig feature)\n  - \"ss01\": Enables OpenType stylistic set 1. Can enable sets 1 to 10 by changing the last two digits.\nWindows-only options:\n  - \"directwrite\": (default) Use DirectWrite for font rendering\n  - \"gdi\": Use GDI for font rendering\n  - \"dwrite_cleartype_classic\": Only applicable to \"directwrite\" mode, should render fonts similar to traditional GDI\n  - \"dwrite_cleartype_natural\": Only applicable to \"directwrite\" mode, should render fonts similar to ClearType \"natural quality\" GDI\n  - \"subpixel_antialias\": Force ClearType antialiasing when disabled at system level\nMac-only options:\n  - \"no_round\": Don't round glyph widths for monospace fonts when `font_size` is small.",
+      "type": "array",
+      "uniqueItems": true,
+      "items": {
+        "type": "string",
+        "enum": [
+          "no_bold",
+          "no_italic",
+          "no_antialias",
+          "gray_antialias",
+          "no_liga",
+          "no_clig",
+          "no_calt",
+          "dlig",
+          "ss01",
+          "ss02",
+          "ss03",
+          "ss04",
+          "ss05",
+          "ss06",
+          "ss07",
+          "ss08",
+          "ss09",
+          "ss10",
+          "directwrite",
+          "gdi",
+          "dwrite_cleartype_classic",
+          "dwrite_cleartype_natural",
+          "subpixel_antialias",
+          "no_round"
+        ]
```

Comment:
```suggestion
        ],
        "enumDescriptions": [
          "Disable bold text.",
	  "Disable italic text.",
	  "Disable antialiasing.",
	  "Use grayscale antialiasing instead of subpixel.",
	  "Disable standard ligatures (OpenType liga feature).",
	  "Disable contextual ligatures (OpenType clig feature).",
	  "Disable contextual alternatives (OpenType calt feature).",
	  "Enable discretionary ligatures (OpenType dlig feature).",
	  "Enable OpenType stylistic set 1.",
	  "Enable OpenType stylistic set 2.",
	  "Enable OpenType stylistic set 3.",
	  "Enable OpenType stylistic set 4.",
	  "Enable OpenType stylistic set 5.",
	  "Enable OpenType stylistic set 6.",
	  "Enable OpenType stylistic set 7.",
	  "Enable OpenType stylistic set 8.",
	  "Enable OpenType stylistic set 9.",
	  "Enable OpenType stylistic set 10.",
	  "Use DirectWrite for font rendering (default, Windows only!).",
	  "Use GDI for font rendering (Windows only!).",
	  "Only applicable to \"directwrite\" mode, should render fonts similar to traditional GDI (Windows only!).",
	  "Only applicable to \"directwrite\" mode, should render fonts similar to ClearType \"natural quality\" GDI (Windows only!).",
	  "Force ClearType antialiasing when disabled at system level (Windows only!).",
	  "Don't round glyph widths for monospace fonts when `font_size` is small (Mac only!)."
    ]
```